tests/util/test_domain.py

```diff
@@ -54,10 +54,25 @@ def test_create_coord():
 )
 def test_convert_longitude(lon_in, expected_lon_out):
     """Test expected longitude conversion for individual values and range."""
-    lon_out = convert_longitude(lon_in)
+    lon_out = convert_lon_to_180(lon_in)
     np.testing.assert_allclose(lon_out, expected_lon_out)
 
 
+@pytest.mark.parametrize(
+    "lon_in,expected_lon_out",
+    [
+        (np.array([360.0]), np.array([0.0])),
+        (np.array([181.0]), np.array([181.0])),
+        (np.array([-180.0, 0.0, 179.9]), np.array([180.0, 0.0, 179.9])),
+        (np.arange(1, 361, 1), np.arange(1, 361, 1)),
+    ],
+)
+def test_convert_longitude_scale(lon_in, expected_lon_out):
+    # Your conversion logic here
+    lon_out = convert_lon_to_360(lon_in)
+    assert(lon_out, expected_lon_out)
```

Comment:
```suggestion
    np.testing.assert_allclose(lon_out, expected_lon_out)
```
I think `assert(x, y)` will just return true if both `x` and `y` evaluate to true?

Reply:
done